client/cli/src/params.rs

```diff
@@ -491,6 +509,20 @@ pub struct RunCmd {
 	#[structopt(long = "force-authoring")]
 	pub force_authoring: bool,
 
+	/// Comma separated list of targets for tracing
+	#[structopt(long = "tracing-targets")]
+	pub tracing_targets: Option<String>,
```

Comment:
I think you should add `value_name = "TARGETS"` so it look consistent with other flags.

Reply:
done